keda/2.13.1/manifests/keda.sh_kedacontrollers.yaml

```diff
@@ -3595,14 +3595,6 @@ spec:
                     items:
                       type: string
                     type: array
-                  caConfigMaps:
```

Comment:
This _seems_ fine because we're never going to ship 2.13.1 off of this branch again, but if you want me to stuff this back in a permanent carry patch I will.